### ISSUE TITLE
TimeoutError: timed out Bedrock Server

### ISSUE THREAD

**Issue by ihs4ncim** [py-mine/mcstatus#483]
[ERROR   ] discord.client: Ignoring exception in on_ready
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\site-packages\discord\client.py", line 409, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\Administrator\Desktop\magic\bot.py", line 29, in on_ready
    status = server.status()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\site-packages\mcstatus\utils.py", line 66, in sync_wrapper
    raise last_exc  # type: ignore # (This won't actually be unbound)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\site-packages\mcstatus\utils.py", line 62, in sync_wrapper
    return func(*args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\site-packages\mcstatus\server.py", line 213, in status
    return BedrockServerStatus(self.address, self.timeout, **kwargs).read_status()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\site-packages\mcstatus\bedrock_status.py", line 49, in read_status
    data = self._read_status()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\site-packages\mcstatus\bedrock_status.py", line 58, in _read_status
    data, _ = s.recvfrom(2048)
TimeoutError: timed out``

**Comment by ihs4ncim** [comment]
Code: 
```python
server = BedrockServer.lookup("play.magicyour.tk:19132")
status = server.status()
await bot.change_presence(activity=discord.Game(name=f"{status.players_online} kişi sunucuda"))
```

**Comment by ItsDrike** [comment]
TimeoutError means the connection wasn't established within some specified time. The server you're pinging is likely just offline

**Comment by ihs4ncim** [comment]
> TimeoutError means the connection wasn't established within some specified time. The server you're pinging is likely just offline

n-no, server online.

**Comment by ihs4ncim** [comment]
![image](https://user-images.githubusercontent.com/96793944/217971327-bc6a88cb-0f14-46e6-8c47-ac9c12aecd06.png)

**Comment by ihs4ncim** [comment]
is online

**Comment by ItsDrike** [comment]
Well, a timeout error indicates that the connection couldn't be established within some maximum time. Perhaps there's a firewall blocking the connection, either on your side, or the server may be blocking your IP, it's hard to say. Does this only happen sometimes, or on every attempt? Maybe try pinging that IP, see if you can connect to that IP.

**Comment by PerchunPak** [comment]
The server is active if check with [mcsrvstat.us](https://mcsrvstat.us/bedrock/play.magicyour.tk:19132).

**Comment by ihs4ncim** [comment]
![Screenshot_2023-02-10-11-28-27-40.jpg](https://user-images.githubusercontent.com/96793944/218041597-0df7c1ca-5c97-4999-bf96-b31fde1bb1cf.jpg)

Look

**Comment by PerchunPak** [comment]
Duplicate of #433. Fix for that issue is not on PyPI yet, so you need to install `mcstatus` from 22788ad5d17b2da5d2e04fe4f79df1cd1aa3caf4 (or never, but note that there are a lot of breaking changes in master, which is blocked for releasing by broken documentation).

```bash
pip install git+https://github.com/py-mine/mcstatus.git@22788ad5d17b2da5d2e04fe4f79df1cd1aa3caf4
```

**Comment by ihs4ncim** [comment]
```WARNING: Ignoring invalid distribution -ip (c:\users\administrator\appdata\local\programs\python\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -ip (c:\users\administrator\appdata\local\programs\python\python310\lib\site-packages)
Collecting git+https://github.com/py-mine/mcstatus.git@22788ad5d17b2da5d2e04fe4f79df1cd1aa3caf4
  Cloning https://github.com/py-mine/mcstatus.git (to revision 22788ad5d17b2da5d2e04fe4f79df1cd1aa3caf4) to c:\users\administrator\appdata\local\temp\2\pip-req-build-b3uy0qod
  ERROR: Error [WinError 2] The system cannot find the file specified while executing command git version
ERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?
WARNING: Ignoring invalid distribution -ip (c:\users\administrator\appdata\local\programs\python\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -ip (c:\users\administrator\appdata\local\programs\python\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -ip (c:\users\administrator\appdata\local\programs\python\python310\lib\site-packages)```

**Comment by kevinkjt2000** [comment]
You would have to have `git` installed for that path to work. It's in the ERROR message...
This was already marked as a duplicate. I would suggest moving further discussion into #433, but that issue is already marked as resolved as of Dec 24, 2022 by code from #439. A new release for that version has not been cut yet.

It should be possible to use that same commit-ref without needing git installed by using this URL instead https://github.com/py-mine/mcstatus/archive/22788ad5d17b2da5d2e04fe4f79df1cd1aa3caf4.zip

Locking this conversation at this point.
